### PR TITLE
feature: add news section and reformat home page

### PR DESCRIPTION
**Pull request by natedingyifeng**
@UniverseFly @ganler

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -25,6 +25,14 @@ Workshop(
   </picture>
 </p>
 
+We are pleased to announce that the 1st International Workshop on Large Language Models for Code (**LLM4Code**) will take place in Lisbon, co-located with [ICSE 2024](https://conf.researchr.org/home/icse-2024)! The submission deadline is **Dec 7th**, 2023. The link to submit is [here](https://llm4code2024.hotcrp.com/).
+
+## News
+
+- **Oct 11, 2023** We have launched our Twitter account ([@llm4code](https://twitter.com/llm4code)) with official hashtag **#LLM4Code**!
```
Comment (ganler) [inline index.md]:
```suggestion
- **Oct 11, 2023** Follow us on Twitter/X ([@llm4code](https://twitter.com/llm4code)) to hear timely updates!
```

File: index.md [hunk 2]
```diff
@@ -25,6 +25,14 @@ Workshop(
   </picture>
 </p>
 
+We are pleased to announce that the 1st International Workshop on Large Language Models for Code (**LLM4Code**) will take place in Lisbon, co-located with [ICSE 2024](https://conf.researchr.org/home/icse-2024)! The submission deadline is **Dec 7th**, 2023. The link to submit is [here](https://llm4code2024.hotcrp.com/).
```
Comment (ganler) [inline index.md]: Let's remove this line and embed this into news.

File: index.md [hunk 3]
```diff
@@ -25,6 +25,14 @@ Workshop(
...
+
+## News
+
```
Comment (ganler) [inline index.md]:
```suggestion
- **Oct 11, 2023** Our [HotCRP site](https://llm4code2024.hotcrp.com/) is ready for your submissions!
```

**Review by ganler — CHANGES_REQUESTED** [review]

File: index.md (hunk 2 above)
Comment (UniverseFly) [inline index.md]:
```suggestion
We are pleased to announce that the 1st International Workshop on Large Language Models for Code (**LLM4Code**) will take place in Lisbon, co-located with [ICSE 2024](https://conf.researchr.org/home/icse-2024)! The submission deadline is **Dec 7th**, 2023. You can submit through [this link](https://llm4code2024.hotcrp.com/).
```

**Review by UniverseFly — APPROVED** [review]

File: index.md [hunk 4]
```diff
@@ -25,6 +25,14 @@ Workshop(
   </picture>
 </p>
 
+We are pleased to announce that the 1st International Workshop on Large Language Models for Code (**LLM4Code**) will take place in Lisbon, co-located with [ICSE 2024](https://conf.researchr.org/home/icse-2024)! The submission deadline is **Dec 7th**, 2023. The link to submit is [here](https://llm4code2024.hotcrp.com/).
+
+## News
+- **Oct 11, 2023** Our [HotCRP site](https://llm4code2024.hotcrp.com/) is ready for your submissions!
```
Comment (ganler) [inline index.md]: add some quick deadline info

**Review by ganler — CHANGES_REQUESTED** [review]

File: index.md (hunk 4 above)
Comment (natedingyifeng) [reply index.md]: Added

**Review by ganler — APPROVED** [review]

**Pull request merged**